### DISCUSSION TITLE
Precedence Diagrams

### DISCUSSION THREAD

**Discussion by aure30** [Hoogkamer/Termboard#22] [Q&A] [answered]
I have more than 200 criteria embodied in 52 goals. How can I quickly and accurately create and display the 4 predecessor-successor relationships? i.e start to start, start to finish, finish to start, and finish to finish?

**Comment by Hoogkamer** [comment, marked as answer]
You can use excel to list the names for the criteria and goals in the terms tab, and relations (for the dependencies) in the relations tab.
For example relation will be 3 columns with source, relation, target (goal1, has criteria, criteria1) and/or (goal1, depends on, goal2). And then import it: https://youtu.be/clCKmkpFldA?feature=shared&t=94  Does this help?
